includes/admin/class-admin-tools.php

```diff
@@ -16,7 +16,8 @@ class WeForms_Admin_Tools {
      */
     public static function import_json_file( $file ) {
         $encode_data = file_get_contents( $file );
-        $options     = json_decode( $encode_data, true );
+        $jsonData = html_entity_decode($encode_data);
+        $options = json_decode( $jsonData, true );
```

Comment:
Equal signs not aligned
Needs spacing within parentheses

Reply:
please check now